CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.3.0
+- Thanks to Bill Haggerty for their contribution of adding proper component removal
```

Comment:
```suggestion
```

imho this should not be in a CHANGELOG, while we do appreciate each and every contributor we like to have the GIT commits and their authors speak for themselves.